### ISSUE TITLE
[javascript] Create marketing.md

### ISSUE THREAD

**Issue by karlalopez** [gdisf/teaching-materials#456]
1. Create `marketing.md` in the lesson folder
2. Add content:

```
## One Paragraph Announcement

:tada: * JS101: Intro to JavaScript
* :tada:
:calendar: Calendar Day, Month
:clock6: XX AM - XX PM @ Venue

JavaScript is the programming language that makes web pages interactive. It is also used to make servers, robots, and more! That makes it one of the most useful first languages to learn.
Prerequisites: Basic HTML & CSS

We also have scholarships available.

*Sign up here:* [TODO: add link to meetup]

## Example Tweets
- Join us for our #Javascript workshops and learn how to make your website more interactive!  [LINK to meetup]
- New to web development ? Come to our 2-Day Intro to JavaScript this weekend where you'll learn the fundamentals of programming in #JavaScript [LINK to meetup]
- Ready to learn JavaScript? @gdisf has a weekend workshop. No programming experience necessary! [LINK to meetup]
- Open spots for Intro to JavaScript (2 days) with @gdisf [LINK to meetup]



```

**Comment by snidelyhazel** [comment]
I got this!

**Comment by verythorough** [comment]
Closed by #535. Thank you!
